### PR TITLE
Check if a session exists before regenerating

### PR DESCRIPTION
**Pull request by driesdroesbeke**

### CODE REVIEW

File: library/Zend/Session.php [hunk 1]
```diff
@@ -377,7 +377,9 @@ public static function rememberUntil($seconds = 0)
             );
 
         // normally "rememberMe()" represents a security context change, so should use new session id
-        self::regenerateId();
+        if(self::sessionExists()){
```
Comment (mhujer) [inline library/Zend/Session.php]: there should be space after `if`

File: library/Zend/Session.php [hunk 2]
```diff
@@ -377,7 +377,9 @@ public static function rememberUntil($seconds = 0)
...
+        	self::regenerateId();
```
Comment (mhujer) [inline library/Zend/Session.php]: Please use 4 spaces instead of tabs

**Comment by mhujer** [comment]
@driesdroesbeke can you please add a test for this case?

**Comment by driesdroesbeke** [comment]
@mhujer sorry, I used the online editor and forgot about the spaces.

I added a test for this case. Without the fix this test works on PHP 5.X but not on PHP7.

**Comment by froschdesign** [comment]
@driesdroesbeke 
Sorry, but I can not found signed [CLA](https://github.com/zendframework/zf1/wiki/Contributor-License-Agreement-%28CLA%29).

**Comment by jtnix** [comment]
I am depressed that this 1-line patch, crucial for full custom session handler support with ZF1 and PHP 7, is being held up due to lack of signed CLA and unit test nit-pickery. 

If I recommit the unit test with the missing code with preferred 'Array' suffix notation and sign a CLA can we get this thing going again?

**Comment by froschdesign** [comment]
@jtnix 
> …is being held up due to lack of signed CLA

These are the rules no matter if the contributed code has 1 or 10.000 lines.

> …and unit test nit-pickery.

Really simple: the test (assertion) is missing.

> …crucial for full custom session handler support with ZF1 and PHP 7

Please keep in mind: ZF1 has no official support for PHP 7.

> …can we get this thing going again?

Create a PR and we will review it.

**Comment by froschdesign** [comment]
Closing due inactivity.

Please feel free to open a new PR. We will review it.

**Comment by githoober** [comment]
How is this issue invalid?

**Comment by froschdesign** [comment]
@githoober 

1. Unit test was missing
3. No new PR was provided

**Pull request closed without merging**